MekHQ/src/mekhq/gui/utilities/JMoneyTextField.java

```diff
@@ -53,14 +55,14 @@ public JMoneyTextField(mekhq.Action whenEnterPressed) {
 
     public Money getMoney() {
         try {
-            return Money.of(Double.parseDouble(this.getValue().toString()));
+            return Money.of(this.format.parse(this.getText()).doubleValue());
         } catch (Exception ignored) {
             return Money.zero();
         }
     }
 
     public void setMoney(Money money) {
-        assert money != null;
+       assert money != null;
```

Comment:
This seems odd.

Reply:
Why?

Reply:
The whitespace change seemed unnecessary.

Reply:
I'll double check I have tabs as spaces, it may be IDEA autoformatting stuff.

Reply:
Mmm, one whitespace disappeared, great catch!